giraffe/src/components/SimpleTable/PagedTable.tsx

```diff
@@ -54,31 +56,28 @@ const getNumberOfRowsOnCurrentPage = (
   let lastSignature
   let signature
 
-  // rowHeight is now guaranteed to be greater than zero
-  const lastVisibleRowMinimumHeight = 0.2 * rowHeight
-
   while (rowIdx < result.table.length) {
     if (result.table.columns?.table?.data?.[rowIdx] !== currentTable) {
       signature = Object.values(result.table.columns)
-        .map(
-          c =>
-            `${c.name}::${c.fluxDataType}::${result.fluxGroupKeyUnion.includes(
-              c.name
-            )}`
-        )
+        .map(column => {
+          if (column.data[rowIdx] !== undefined) {
+            return `${column.name}::${
+              column.fluxDataType
+            }::${result.fluxGroupKeyUnion.includes(column.name)}`
+          }
+          return 'undefined'
```

Comment:
What's this portion of the code - returning the string `undefined` - doing?

Reply:
@wdoconnell it's building a 'signature' (what actually seems like a [composite key](https://en.wikipedia.org/wiki/Composite_key)) based on the `name`, `fluxDataType` and `fluxGroupUnionKey`. It looks like the previous code incorrectly didn't account for when `column.data[rowIdx]` didn't exist

Reply:
Excellent explanation. Thanks @hoorayimhelping